core/extensions/AssemblyLoading/AssemblyLoading/AssemblyLoading.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="../MyLibrary/MyLibrary.csproj">
```

Comment:
Maybe for the sake of the sample - add a comment which describes in human readable form what these various options do in this case (MSBuild and our use of it in this case is rather cryptic).

Reply:
Good call.